### ISSUE TITLE
next and previous page links don't go where I'd expect

### ISSUE THREAD

**Issue by Aariq** [eliocamp/reproducibility-with-r#25]
At the bottom of the Day 0 page, the "next page" link goes to the mini reprohack page, not Day 1 like I would have expected.  The previous page on the mini reprohack page goes back to Day 0.

On the [main day 1 page](https://reproducibility.rocks/materials/day1/), there is no next or previous page link (this applies to the main page for day 2–4 also)

(part of JOSE review here: https://github.com/openjournals/jose-reviews/issues/260)

**Comment by eliocamp** [comment]
Yeah, this is not ideal. I tried looking into this and it seems to be a limitation with the apero theme we're using since their example site has the same problems. I [opened an issue](https://github.com/hugo-apero/hugo-apero/issues/114) upstream to see if this is something that can be fixed site-side or if it should be fixed on their end.
